tools/rum-models-generator/Tests/rum-models-generator-coreTests/Fixtures/Output/RUMDataModels.swift

```diff
@@ -0,0 +1,1474 @@
+        let size: Int64?
+
+        /// SSL phase properties
+        let ssl: SSL?
+
+        /// HTTP status code of the resource
+        let statusCode: Int64?
+
+        /// Resource type
+        let type: ResourceType
+
+        /// URL of the resource
+        var url: String
+
+        enum CodingKeys: String, CodingKey {
+            case connect = "connect"
+            case dns = "dns"
+            case download = "download"
+            case duration = "duration"
+            case firstByte = "first_byte"
+            case id = "id"
+            case method = "method"
+            case provider = "provider"
+            case redirect = "redirect"
+            case size = "size"
+            case ssl = "ssl"
+            case statusCode = "status_code"
+            case type = "type"
+            case url = "url"
+        }
+
+        /// Connect phase properties
+        internal struct Connect: Codable {
+            /// Duration in ns of the resource connect phase
+            let duration: Int64
+
+            /// Duration in ns between start of the request and start of the connect phase
+            let start: Int64
+
+            enum CodingKeys: String, CodingKey {
+                case duration = "duration"
+                case start = "start"
+            }
+        }
+
+        /// DNS phase properties
+        internal struct DNS: Codable {
+            /// Duration in ns of the resource dns phase
+            let duration: Int64
+
+            /// Duration in ns between start of the request and start of the dns phase
+            let start: Int64
+
+            enum CodingKeys: String, CodingKey {
+                case duration = "duration"
+                case start = "start"
+            }
+        }
+
+        /// Download phase properties
+        internal struct Download: Codable {
+            /// Duration in ns of the resource download phase
+            let duration: Int64
+
+            /// Duration in ns between start of the request and start of the download phase
+            let start: Int64
+
+            enum CodingKeys: String, CodingKey {
+                case duration = "duration"
+                case start = "start"
+            }
+        }
+
+        /// First Byte phase properties
+        internal struct FirstByte: Codable {
+            /// Duration in ns of the resource first byte phase
+            let duration: Int64
+
+            /// Duration in ns between start of the request and start of the first byte phase
+            let start: Int64
+
+            enum CodingKeys: String, CodingKey {
+                case duration = "duration"
+                case start = "start"
+            }
+        }
+
+        /// HTTP method of the resource
+        internal enum Method: String, Codable {
+            case pOST = "POST"
+            case gET = "GET"
+            case hEAD = "HEAD"
+            case pUT = "PUT"
+            case dELETE = "DELETE"
+            case pATCH = "PATCH"
```

Comment:
the first letter should be uppercase or other letters should be lowercase

Reply:
✅